GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -455,12 +456,75 @@ public FileContentResult DownloadUserConsentAllowContactForFeedback()
             return fileContentResult;
         }
 
-        [Route("/download")]
+        [HttpGet("downloads/erhc-all-organisations")]
+        public FileContentResult EhrcAllOrganisationsForYear_AdminPage(int year)
+        {
+            return GenerateEhrcAllOrganisationsForYearFile(year);
+        }
+
+        [Route("/download"
+             /* This is "/download" rather than "download" because the URLs we've shared with the EHRC are of the format:
+              https://gender-pay-gap.service.gov.uk/download?p=App_Data%5CDownloads%5CGPG-Organisations_2017-18.csv
+              just "download" would make the URLs "/admin/download" */
+        )]
         [AllowOnlyTrustedIps(AllowOnlyTrustedIps.IpRangeTypes.EhrcIPRange)]
-        public IActionResult EhrcDownloadOrganisationsForYear(string p)
+        public IActionResult EhrcAllOrganisationsForYear_EhrcIpProtectedLink(string p)
```

Comment:
What is the form of `p` here? Is it a year? Or the filename? Could we make it a bit clearer?